egs/hkust/s5/local/chain/tuning/run_tdnn_2a.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+# Copyright 2018  Emotech LTD (Author: Xuechen Liu)
+
+# This script is based on run_tdnn_7h.sh in swbd chain recipe.
+# exp 2a: change the step of making configs, using xconfig
+#         some minor changes on training parameters, referencing wsj
+
+# Results
+# local/nnet3/compare_wer_general.sh --online exp/chain/tdnn_7h_chain_2asp
```

Comment:
this compare_wer_general.sh script doesn't seem to exist and the results don't seem very consistent.

I'm surprised you took such an old Swbd script as the source.  The latest one probably would have given better results.

Reply:
Thanks for the reply!

I will re-run online decoding for a further check. 

The swbd comment is inherited from old script and I forogt to delete it. The one I modified from is actually run_tdnn_1a.sh in chain/nnet3 directory, which was the one for hkust results.

Speaking of output of chain info:
```
exp/chain/tdnn_7h_sp: num-iters=435 nj=4..4 num-params=11.8M dim=43+100->4000 combine=-0.100->-0.100 (over 9) xent:train/valid[289,434,final]=(-1.79,-1.76,-1.75/-1.72,-1.71,-1.71) logprob:train/valid[289,434,final]=(-0.119,-0.107,-0.107/-0.153,-0.155,-0.154)
```
(the one above is actually 'chain_2a')
```
exp/chain/tdnn_7h_chain_2b_sp: num-iters=435 nj=3..4 num-params=11.8M dim=43+100->4000 combine=-0.114->-0.114 (over 3) xent:train/valid[289,434,final]=(-1.74,-1.71,-1.71/-1.95,-1.96,-1.95) logprob:train/valid[289,434,final]=(-0.124,-0.114,-0.114/-0.172,-0.177,-0.176)
```